src/web/routes/auth/auth-router.ts

```diff
@@ -21,8 +21,8 @@ authRouter.use(jiraServerSymmetricJwtAuthMiddleware);
  */
 authRouter.get(
 	['/checkAuth'],
+	requestSchemaValidationMiddleware(CHECK_AUTH_REQUEST_SCHEMA),
 	function (req: CheckAuthRequest, res: CheckAuthResponse, next: NextFunction) {
-		assertSchema(req.query, CHECK_AUTH_QUERY_PARAMETERS_SCHEMA);
```

Comment:
**note:** Use `requestSchemaValidationMiddleware` to validate a request. This allows to:

- Validate a `req` schema before declaring `req` of a specific type and using it. 
- Encapsulate the concern and simplify route handlers.

Reply:
Very nice